wgsl/index.bs

```diff
@@ -4157,13 +4320,33 @@ use the same memory.
 
 # Expressions # {#expressions}
 
-Expressions specify how values are computed.
+[=Expressions=] specify how values are computed.
+
+The different kinds of value expressions provide a tradeoff between when they
+are evaluated and how expressive they can be.
+The sooner the evaluation, the more constrained the operations, but also the
+more places the value can be used.  This tradeoff leads to different
+flexibility with each kind of value declaration.
+[=const-expressions=] and [=override-expressions=] are evaluated prior to
+execution on the GPU, so only the result of the computation of the expression
+is necessary in the final GPU code.
+Additionally, because [=const-expressions=] are evaluated at [=shader module
+creation|shader-creation time=] they can be used in more situations than
+[=override-expressions=], for example, to size [=arrays=] in [=function scope=]
+[=variable declaration|variables=].
+<dfn noexport>Runtime expressions</dfn> on the other hand, are computed
+completely on the GPU.
```

Comment:
I would delete "completely".

The computation completes on the GPU, but that's not the same thing.